README.md

```diff
@@ -1197,6 +1197,22 @@ require('@digitalocean/do-markdownit/util/prism_keep_html')(Prism);
 Prism.highlight('console.log("<mark>Hello, world!</mark>");', Prism.languages.javascript, 'javascript');
 ```
 
+## limit_tokens
+
+<details>
+<summary>Filters and transforms tokens in the token stream.</summary>
+
+**Options:**
+
+Pass options for this plugin as the `limit_tokens` property of the `do-markdownit` plugin options.
+Set this property to `false` to disable this plugin.
+
+- `allowedTokens` (`array`): A list of Markdown tokens that should render
+- `transformTokens` (`object`): An object where the keys are Markdown tokens that should be transformed.
+  The transformation is done based on the value which is an `function` that expects a Markdown token as a param
```

Comment:
```suggestion
  The transformation is done based on the value which is a `function` that expects a [Markdown token](https://markdown-it.github.io/markdown-it/#Token) as a param
```
Probably worth including a link back to the docs here

Reply:
Updated.